tests/barman-simple-backup/run.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+docker-compose down -v && docker-compose build && docker-compose up -d pgmaster backup
+docker-compose exec  -T pgmaster wait_local_postgres
+
+echo ">>> Create a test table and fill some data"
+docker-compose exec pgmaster psql -U postgres monkey_db -c " create table t (time timestamp);";
+for i in `seq 1 15`; do
+  sleep 1
+  docker-compose exec -T pgmaster psql -U postgres monkey_db -c "insert into t values(now());";
+done
+sleep 10
+echo ">>> Make a backup"
+docker-compose exec -T backup barman switch-xlog --force --archive all
+docker-compose exec -T backup barman backup all
+
+NUM_BACKUPS=`docker-compose exec -T backup barman list-backup all | grep -v 'DEBUG:' | wc -l | tr -d ' '`;
```

Comment:
Just a suggestion:
replace `grep` with `barman list-backup all 2>/dev/null`

Reply:
```
└$ docker-compose exec -T backup barman list-backup all 2>/dev/null
2017-09-06 15:50:09,464 [64] barman.config DEBUG: Including configuration file: upstream.conf
2017-09-06 15:50:09,465 [64] barman.cli DEBUG: Initialised Barman version 2.2 (config: /etc/barman.conf, args: {'command': 'list_backup', 'server_name': ['all'], 'format': 'console', 'debug': False, 'quiet': False, 'minimal': False})
pg_cluster 20170906T154956 - Wed Sep  6 15:49:57 2017 - Size: 35.1 MiB - WAL Size: 0 B
```

debug is still there